interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -41,9 +58,61 @@ components:
           type: string
           example: v0.0.1-445ad3
 
+    GeozoneSourceRequest:
```

Comment:
nit: I'd suggest `CreateGeozoneSourceRequest` since just `GeozoneSourceRequest` sounds like maybe we're using this to request a GeozoneSource (we're actually doing the opposite and trying to make one rather than get one)

Reply:
Sounds very good, thanks